### PR TITLE
[8.0] Add l10n_it_central_journal

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

File: l10n_it_central_journal/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Diego Milani
+* Davide Corio
+* Lorenzo Battistini
```
Comment (tafaRU) [inline l10n_it_central_journal/README.rst]: Please add email addresses.

**Comment by tafaRU** [comment]
Please use short headers as in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

File: l10n_it_central_journal/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+{
+    'name': 'Italian Localization - Account central journal',
+    'version': '8.0.2.0.0',
+    'category': 'Localization/Italy',
+    'author': 'Link It Spa',
```
Comment (tafaRU) [inline l10n_it_central_journal/__openerp__.py]: Please add OCA as author.

**Comment by eLBati** [comment]
@tafaRU thanks , done

**Comment by eLBati** [comment]
:green_heart:

File: l10n_it_central_journal/README.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+==============
+Libro giornale
+==============
+
+Modulo per la stampa del libro giornale
+
+Usage
+=====
+
+* Go to ...
```
Comment (tafaRU) [inline l10n_it_central_journal/README.rst]: Please remove it

**Comment by zeroincombenze** [comment]
Come faccio a caricare le modifiche per provarle?

**Comment by eLBati** [comment]
Per provarle nel tuo repo locale di l10n-italy
```
git remote add eLBati git://github.com/eLBati/l10n-italy.git
git fetch eLBati
git checkout add_libro_giornale
```

Altrimenti su runbot:
https://odoo-community.org/page/review

**Comment by OpenCode** [comment]
:+1: It works

**Comment by scigghia** [comment]
:+1:

**Pull request merged**